.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: build
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - 2.7
+          - 2.6
+        gemfile:
+          #- rails_4.2.gemfile
+          #- rails_4.2_sprockets4.gemfile
+          - rails_5.0.gemfile
+          - rails_5.0_sprockets4.gemfile
+          - rails_5.2.gemfile
+          - rails_5.2_sprockets4.gemfile
+          - rails_6.0.gemfile
+          - rails_6.1.gemfile
+          - rails_7.0.gemfile
+          #- rails_head.gemfile
+          #- rails_head_sprockets4.gemfile
```

Comment:
I have rails_head and rails 4.2 enabled back in a different branch... I wasn't sure if you want to review separately.

4.2 is a no-brainer, ...

rails_heads needs to allow rails 7.1